lib/src/parse/parser.dart

```diff
@@ -133,7 +133,8 @@ class Parser {
     whitespace();
   }
 
-  /// Consumes and ignores a silent (Sass-style) comment.
+  /// Consumes and ignores a single silent (Sass-style) comment, not including
+  /// the trailing newline.
```

Comment:
"not including the trailing newline" makes me wonder about what happens when there are consecutive lines with silent comments, but otherwise LGTM

Reply:
This function will only consume one of them, but `whitespace()` will consume them all _and_ the whitespace.